products/compute/api.yaml

```
@@ -1329,43 +1548,238 @@ objects:
               use the timeout of 10 min (currently fixed). Setting to false has the
               following effect: both old and new connections will have a drain timeout
               of 10 min.
-
               This can be set to true only if the protocol is TCP.
-
               The default is false.
           - !ruby/object:Api::Type::Boolean
             name: 'dropTrafficIfUnhealthy'
             description: |
               This option is used only when no healthy VMs are detected in the primary
               and backup instance groups. When set to true, traffic is dropped. When
               set to false, new connections are sent across all VMs in the primary group.
-
               The default is false.
           - !ruby/object:Api::Type::Double
             name: 'failoverRatio'
             description: |
               The value of the field must be in [0, 1]. If the ratio of the healthy
               VMs in the primary backend is at or below this number, traffic arriving
               at the load-balanced IP will be directed to the failover backend.
-
               In case where 'failoverRatio' is not set or all the VMs in the backup
               backend are unhealthy, the traffic will be directed back to the primary
               backend in the "force" mode, where traffic will be spread to the healthy
               VMs with the best effort, or to all VMs when no VM is healthy.
-
               This field is only used with l4 load balancing.
       - !ruby/object:Api::Type::Fingerprint
         name: 'fingerprint'
         output: true
         description: |
           Fingerprint of this resource. A hash of the contents stored in this
           object. This field is used in optimistic locking.
+      - !ruby/object:Api::Type::Array
+        name: 'healthChecks'
+        item_type: Api::Type::String
+        required: true
+        min_size: 1
+        max_size: 1
+        description: |
+           The set of URLs to HealthCheck resources for health checking
+           this RegionBackendService. Currently at most one health
+           check can be specified, and a health check is required.
+      - !ruby/object:Api::Type::Integer
+        name: 'id'
+        description: 'The unique identifier for the resource.'
+        output: true
       - !ruby/object:Api::Type::Enum
-        name: 'protocol'
+        name: 'loadBalancingScheme'
+        input: true
         description: |
-          The protocol this BackendService uses to communicate with backends.
+          Indicates whether the backend service will be used with internal or
+          external load balancing. A backend service created for one type of
+          load balancing cannot be used with the other. Must be `INTERNAL`
+          or `INTERNAL_MANAGED` for a regional backend service. Defaults to `INTERNAL`.
```

```suggestion
          or `INTERNAL_MANAGED`. Defaults to `INTERNAL`.
```

nit: This description is specific to an internal backend service.